flexbox/src/main/java/com/google/android/flexbox/FlexboxLayout.java

```diff
@@ -497,7 +497,7 @@ private void measureHorizontal(int widthMeasureSpec, int heightMeasureSpec) {
                         child.getMeasuredHeight() + lp.topMargin + lp.bottomMargin);
 
                 if (isWrapRequired(mFlexWrap, widthMode, widthSize, flexLine.mainSize,
```

Comment:
I think you don't have to add an argument to the `isWrapRequired` method, instead you can change the fourth argument as `child.getMeasuredWidth + lp.leftMargin + lp.rightMargin`.